source/elements/oneTBB/source/task_scheduler/task_group/task_group_cls.rst

```diff
@@ -44,6 +46,10 @@ Member functions
 
     Constructs an empty ``task_group``.
 
+.. cpp:function:: task_group(task_group_context& context)
+
+    Constructs an empty ``task_group``, which tasks are associated with the ``context``.
```

Comment:
```suggestion
    Constructs an empty ``task_group``, which associates all tasks created with `run` and `run_and_wait` methods with with the ``context``.
```

Reply:
IMHO mentioning all methods which adds a new task into the `task_group` is redundant (given that `defer` is being added in a parallel PR).

what about this : 

>   Constructs an empty ``task_group``. All tasks added in into ``task_group`` are associated with the ``context``.

?

Reply:
The last version is fine, fix a typo though: 
```suggestion
    Constructs an empty ``task_group``. All tasks added into the ``task_group`` are associated with the ``context``.
 ```